tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -105,39 +105,39 @@
+    <string name="pref_fast_dns_server">DNSCrypt Servers</string>
+    <string name="pref_fast_select_dns_server">Selected:</string>
     <string name="pref_fast_tor">Tor Settings</string>
     <string name="pref_fast_routing">Routing Settings</string>
     <string name="pref_fast_all_through_tor">Route All traffic through Tor</string>
     <string name="pref_fast_all_through_ipro">Route All traffic through InviZible</string>
-    <string name="pref_fast_unlock_with_tor">Select WebSites</string>
-    <string name="pref_fast_unlock_with_tor_summ">WebSites list to open with Tor. This feature does not work for sites behind CDN</string>
-    <string name="pref_fast_unlock_apps_with_tor">Select Applications</string>
-    <string name="pref_fast_unlock_apps_with_tor_summ">Applications list to use with Tor</string>
-    <string name="pref_fast_unlock_apps_with_ipro_summ">Applications list to use with InviZible</string>
-    <string name="pref_fast_exclude_from_tor">Exclude WebSites</string>
-    <string name="pref_fast_exclude_from_tor_summ">WebSites list to open directly. This feature does not work for sites behind CDN</string>
-    <string name="pref_fast_exclude_apps_from_tor">Exclude Applications</string>
-    <string name="pref_fast_exclude_apps_from_tor_summ">Applications list to open directly</string>
+    <string name="pref_fast_unlock_with_tor">Select Websites</string>
+    <string name="pref_fast_unlock_with_tor_summ">List of websites to open with Tor. (Does not work for websites behind CDNs.)</string>
+    <string name="pref_fast_unlock_apps_with_tor">Select Apps</string>
+    <string name="pref_fast_unlock_apps_with_tor_summ">List of apps to use with Tor.</string>
+    <string name="pref_fast_unlock_apps_with_ipro_summ">List of apps use with InviZible.</string>
+    <string name="pref_fast_exclude_from_tor">Excluded Webstes</string>
+    <string name="pref_fast_exclude_from_tor_summ">List of websites to open directly. (Does not work for websites behind CDNs.)</string>
+    <string name="pref_fast_exclude_apps_from_tor">Excluded Apps</string>
+    <string name="pref_fast_exclude_apps_from_tor_summ">List of apps to open directly.</string>
     <string name="pref_fast_site_refresh_interval">Refresh Interval</string>
-    <string name="pref_fast_site_refresh_interval_summ">Period in hours to refresh WebSites IPs. For Android 5.1 and higher. Put 0 to stop refresh.</string>
-    <string name="pref_fast_use_tor_bridges">Bridges</string>
-    <string name="pref_fast_use_tor_bridges_summ">Use it if you can not connect to Tor network</string>
-    <string name="pref_fast_use_tor_bridges_alt_summ">Please disable Entry Nodes in Tor Settings, otherwise you cannot use Bridges.</string>
+    <string name="pref_fast_site_refresh_interval_summ">Amount of hours between refreshing website IPs (for Android 5.1 and higher). Use \"0\" to prevent refreshing.</string>
```

Comment:
```suggestion
    <string name="pref_fast_site_refresh_interval_summ">Amount of hours between refreshing website IPs (for Android 5.1 and higher). Use \"0\" to prevent it.</string>
```